docs/TuxActions.md

```diff
@@ -145,8 +152,8 @@ This property is attached to each action verb method and represents the full nam
     console.assert(actionCategory.add.type === 'todos_add');
 ```
 
-#### <a id="actionCategory-before"></a>Method - `actionCategory.before` - type: FUNCTION [#](#actionCategory-before)
-Accepts an action verb string or array of action verbs and a callback, it invokes the callback before dispatching the action when the action method is invoked. The major use case for the `before` callback is to allow actions to invoke async functions (such as Ajax requests) and then dispatch the results of those requests.
+#### <a id="actionCategory-before"></a>5.3) Method - `actionCategory.before` - type: FUNCTION [#](#actionCategory-before)
+Accepts an action verb string or array of action verbs and a callback, it invokes the callback before dispatching the action when the action method is invoked. The major use case for the `before` callback is to allow actions to invoke async functions (such as Ajax requests) and then dispatch the results of those requests. `before` will throw an error if it cannot find the passed in `actionVerb` in the `actionCategory`.
```

Comment:
Recommend getting a capital word to start the last sentence instead of `before`.